### ISSUE TITLE
Table caption is not accessible via keyboard navigation

### ISSUE THREAD

**Issue by mabryl** [ckeditor/ckeditor5#16438]
## 📝 Provide detailed reproduction steps (if any)

1. Create a table with a caption
2. Try accessing the caption just by using keyboard navigation

### ✔️ Expected result

You should be able to access the caption element just with the keyboard.

### ❌ Actual result

It's impossible to access the caption by pressing `Tab`, `Shift + Tab` or by using the arrow keys.

## 📃 Other details

* Browser: cross-browser
* OS: cross-OS
* First affected CKEditor version: N/A
* Installed CKEditor plugins: N/A

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mabryl** [comment]
Possibly related to #11834
